templates/etc/init.d/rpm/td-agent

```diff
@@ -148,15 +146,15 @@ if [ -n "${TD_AGENT_USER}" ]; then
   fi
   mkdir -p "$(dirname "${TD_AGENT_PID_FILE}")"
   chown -R "${TD_AGENT_USER}" "$(dirname "${TD_AGENT_PID_FILE}")"
-  START_STOP_DAEMON_ARGS="${START_STOP_DAEMON_ARGS} --user ${TD_AGENT_USER}"
+  START_STOP_DAEMON_ARGS="${START_STOP_DAEMON_ARGS} -u ${TD_AGENT_USER}"
 fi
 
 if [ -n "${TD_AGENT_GROUP}" ]; then
   if ! getent group | grep -q "^${TD_AGENT_GROUP}:"; then
     echo "$0: group for running ${TD_AGENT_NAME} doesn't exist: ${TD_AGENT_GROUP}" >&2
     exit 1
   fi
-  TD_AGENT_ARGS="${TD_AGENT_ARGS} --group ${TD_AGENT_GROUP}"
+  TD_AGENT_ARGS="${TD_AGENT_ARGS} -g ${TD_AGENT_GROUP}"
```

Comment:
-g replaces --group for simplicity? Same as above for -u v.s. --user

Reply:
Well, we did switch from calling `daemon` (which supports `--user` and `--group`) to calling `start_daemon` (which supports `-u` and `-g`). However, you did catch a bug (thanks): the `-g` argument was supposed to be passed to `start_daemon`, not `td-agent`. Just pushed a commit with a fix.

Reply:
Turns out `-g` doesn't work on CentOS/RHEL, so I've restored the original code.